docs/contributing/forem.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Contributing to Forem
+---
+
+# Contributing to Forem
+
+We expect contributors to abide by our underlying
+[code of conduct](https://dev.to/code-of-conduct). All discussions about this
```

Comment:
I don't think this should block merging, but I do think we should standardize around either `Code of Conduct` or `code of conduct`. Took a quick peek throughout our app, and it's currently inconsistent (which is why it shouldn't block merging). Personally, I prefer `Code of Conduct` (feels more in line with CoC abbreviation), since it is a specific file with a Proper Name™️. Standardizing could be handled in a separate PR, either way.